packages/graphql-yoga/__tests__/error-masking.spec.ts

```diff
@@ -287,7 +286,7 @@ describe('error masking', () => {
         },
       ],
     })
-    expect(response.status).toEqual(200)
+    expect(response.status).toEqual(400)
```

Comment:
Why 400? 400 should be thrown by the client's mistake, no?

Reply:
I made a change to default to 200/400 when throwing instances of `GraphQLError`, and 200/500 when throwing instances of `Error`. The handler will still respect the http extensions (if not from resolvers).

See here:
https://github.com/enisdenjo/graphql-yoga/blob/39fbe6f575a0f7ba1a111bd4eb54da5b1ea074d2/packages/graphql-yoga/src/server.ts#L454-L476